docs/guide/acquiring_data/database.rst

```diff
@@ -357,47 +357,32 @@ cannot be found or is not set.
-    5  2014-04-09 06:00:12    N/A                  AIA_3      17.1    17.1
-    6  2011-09-22 00:00:00    2011-09-22 00:00:00  BIR        N/A     N/A
-    7  N/A                    N/A                  N/A        N/A     N/A
-    8  2002-06-25 10:00:10    N/A                  EIT        19.5    19.5
-    9  2002-02-20 11:06:00    2002-02-20 11:06:43  RHESSI     N/A     N/A
-    10 N/A                    N/A                  N/A        N/A     N/A
-    11 N/A                    N/A                  N/A        N/A     N/A
-    12 N/A                    N/A                  N/A        N/A     N/A
-    13 N/A                    N/A                  N/A        N/A     N/A
-    14 N/A                    N/A                  N/A        N/A     N/A
-    15 N/A                    N/A                  N/A        N/A     N/A
-    16 N/A                    N/A                  N/A        N/A     N/A
-    17 N/A                    N/A                  N/A        N/A     N/A
-    18 N/A                    N/A                  N/A        N/A     N/A
-    19 N/A                    N/A                  N/A        N/A     N/A
-    20 2010-10-16 19:12:18    2010-10-16 19:12:22  RHESSI     N/A     N/A
-    21 N/A                    N/A                  N/A        N/A     N/A
-    22 N/A                    N/A                  N/A        N/A     N/A
-    23 N/A                    N/A                  N/A        N/A     N/A
-    24 2012-10-30 15:30:01    N/A                  AIA_4      9.4     9.4
-    25 2012-01-01 00:16:07    N/A                  SWAP       17.4    17.4
-    26 2011-05-08 00:00:00    2011-05-08 00:00:01  AIA        17.1    17.1
-    27 2011-05-08 00:00:00    2011-05-08 00:00:01  AIA        21.1    21.1
-    28 2011-05-08 00:00:02    2011-05-08 00:00:03  AIA        9.4     9.4
-    29 2011-05-08 00:00:03    2011-05-08 00:00:04  AIA        33.5    33.5
-    30 2012-08-05 00:00:01    2012-08-05 00:00:02  AIA        9.4     9.4
-    31 2012-08-05 00:00:01    2012-08-05 00:00:02  AIA        9.4     9.4
-    32 2012-08-05 00:00:02    2012-08-05 00:00:03  AIA        33.5    33.5
-    33 2012-08-05 00:00:02    2012-08-05 00:00:03  AIA        33.5    33.5
-    34 2013-08-05 00:00:01    2013-08-05 00:00:02  AIA        9.4     9.4
-    35 2013-08-05 00:00:01    2013-08-05 00:00:02  AIA        9.4     9.4
-    36 2013-08-05 00:00:02    2013-08-05 00:00:03  AIA        33.5    33.5
-    37 2013-08-05 00:00:02    2013-08-05 00:00:03  AIA        33.5    33.5
-    38 N/A                    N/A                  EIT        25.0    N/A
+    ...                        'instrument', 'wavemin', 'wavemax']))   # doctest: +NORMALIZE_WHITESPACE +REMOTE_DATA
+     id observation_time_start ...      wavemin            wavemax
+    --- ---------------------- ... ------------------ ------------------
+      1    2011-06-07 06:33:02 ...               17.1               17.1
+      2    2011-06-07 06:33:01 ... 13.100000000000001 13.100000000000001
```

Comment:
Which files are producing these long decimals? That's not good, we have to get showing a meaningful number, not the whole set of decimals.